man/mkosi.1

```diff
@@ -259,6 +259,8 @@ either \[lq]1\[rq], \[lq]yes\[rq], or \[lq]true\[rq] to enable, or
 .TP
 \f[V]Distribution=\f[R]
 Matches against the configured distribution.
+Matches any one of multiple distributions. Separate multiple distribution
+names with \[lq]|\[rq] e.g: \[lq]debian|ubuntu\[rq].
```

Comment:
This file is generated from `mkosi.md` in the project root, we don't force anybody to write raw troff. :) Please make the changes there, so they don't get lost when the file is regenerated.